tests/testthat/comparison-newly-failing.txt

```diff
@@ -0,0 +1,18 @@
+✖ minimal 1.0.0                          ── E: 1-1   | W: 0  +1 | N: 0  +1
```

Comment:
These will fail on platforms where `clisymbols` does not use Unicode characters. You don't need to fix it now, we can do it later.